### ISSUE TITLE
Cricket Shots Detection

### ISSUE THREAD

**Issue by abhisheks008** [abhisheks008/DL-Simplified#215]
### Deep Learning Simplified Repository (Proposing new issue)
:red_circle: **Project Title** : Cricket Shots Detection</br>
:red_circle: **Aim** : Create a DL model which will identify the shots played in the game of cricket.</br>
:red_circle: **Dataset** : https://www.kaggle.com/datasets/aneesh10/cricket-shot-dataset</br>
:red_circle: **Approach** : Try to use 3-4 algorithms to implement the models and compare all the algorithms to find out the best fitted algorithm for the model by checking the accuracy scores. Also do not forget to do a exploratory data analysis before creating any model.

**********************************************************************************
### 📍 **Follow the Guidelines to Contribute in the Project :**
* You need to create a separate folder named as the Project Title.
* Inside that folder, there will be four main components.
   * Images - To store the required images.
   * Dataset - To store the dataset or, information/source about the dataset.
   * Model - To store the machine learning model you've created using the dataset.
   * `requirements.txt` - This file will contain the required packages/libraries to run the project in other machines.
* Inside the `Model` folder, the `README.md` file must be filled up properly, with proper visualizations and conclusions.

***********************************************************************
:red_circle::yellow_circle: **Points to Note :**

- The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.
- "Issue Title" and "PR Title should be the same. Include issue number along with it.
- Follow Contributing Guidelines & Code of Conduct before start Contributing.

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name : 
- GitHub Profile Link : 
- Email ID :
- Participant ID (if applicable):
- Approach for this Project :
- What is your participant role? (Mention the Open Source program)

*************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

**Comment by rafiya2003** [comment]
I would like to work on this, frankly I don't know much but I will learn and do. I am a newbie(first year UG student). I want to learn and grow. please give me the opportunity to learn and experience

**Comment by Abhinavcode13** [comment]
Please assign this issue to me @abhisheks008 I have good knowledge in machine learning and python libraries like NumPy pandas etc

**Comment by abhisheks008** [comment]
Please do not to spam in every single issue. This might cause escalation as you are breaching Code of Conduct and Contribution Guidelines.

@rafiya2003

**Comment by abhisheks008** [comment]
As per the code of conduct and contribution guidelines, one contributor can be assigned one issue at a time only.

@Abhinavcode13

**Comment by Mohit-Raje** [comment]
@abhisheks008  I am interested to work on this project . I have good understanding of deep learning and neural nets also I have made project in it

**Comment by abhisheks008** [comment]
Can you please share your approach for solving this issue?
@Sagar201003

**Comment by pro-utkarshM** [comment]
Hey @abhisheks008,    
    My name is Utkarsh Maurya, Please assign me this,
    GitHub Profile Link : https://github.com/pro-utkarshM
    Email ID : projects.utkarshMaurya@gmail.com
    Participant ID (if applicable): 6a93900e-61be-440e-b02d-637b9801f319
    Approach for this Project :
    1) Data Collection (which is given to us)
    2) Data Preprocessing: Preprocess the labeled data to make it suitable for training a DL model.
    3)Model Selection
    4)Model Training & Model Evaluation
    5)Model Optimization: Fine-tune the model by experimenting with different hyperparameters, architectures, or training techniques to improve its performance.

**Comment by abhisheks008** [comment]
Issue assigned to you @pro-utkarshM

**Comment by the-silent-geek** [comment]
Full name : D. Uday Kiran
GitHub Profile Link: https://github.com/the-silent-geek
Email ID: udaykir3911@gmail.com
Approach for this Project: This can be implemented using pre-trained models like VGG-16, ResNet, and DenseNet.
What is your participant role? social winter of code 23
